src/BookList.js

```diff
@@ -27,13 +32,20 @@ export default function BookList() {
   if (loading) return <div>Loading...</div>;
   if (error) return <p>ERROR</p>;
   return (
-    <List>
-      {data.getBooks.map(({ id, title, author }) => (
-        <ListItem key={id}>
-          <Typography variant='body1' className={classes.title}>{title}</Typography>
-          <Typography variant='subtitle2' color='primary'>{author}</Typography>
-        </ListItem>
-      ))}
-    </List>
+      <>
+        {data.getBooks.map(({ id, title, author, numPages }) => (
+          <Grid container key={id}>
+            <Grid item xs={4} sm={3} className = {classes.space}>
```

Comment:
Can you get rid of the surrounding spaces for `=`?

Reply:
Also the `space` class should be on the container not every individual item.

Reply:
I got rid of the spaces and update the class to be in the container.